### DISCUSSION TITLE
Recommendations on Item Links

### DISCUSSION THREAD

**Discussion by mdugue** [pacocoursey/cmdk#128] [Q&A]
Heyho, I would like to add `Item`s to the command `Command.Group` that behave like **links**. Is that possible somehow or are there any examples on the usage with react-router, next router or similar or even plain `<a href="" />`? 

Of course I can attach an imperative `navigate()` call to the `onClick`-handler, but that makes me loose native link benefits, such as open in new tab, previews on iOS, debugging hrefs etc. 

Any Hints or ideas?

**Comment by mdugue** [comment]
@pacocoursey could you leave a short note on wether this is not intended at all, there is already a way to solve it, a PR would be welcome, … ?

**Comment by theromanticsoul** [comment]
+1
